static/app/components/core/menuListItem/index.tsx

```diff
@@ -313,8 +312,12 @@ export const InnerWrap = withChonk(
     font-size: ${p => p.theme.form[p.size ?? 'md'].fontSize};
 
     &,
-    &:hover {
+    &:hover,
+    &:focus,
+    &:focus-visible {
       color: ${getTextColor};
+      box-shadow: none;
```

Comment:
These overrides are necessary because InnerWrap now gets focus instead of MenuItemWrap

Reply:
please make sure you do the same changes in the `chonk` version if they are needed there as well:

https://github.com/getsentry/sentry/blob/e707e1510f02fb654f277910649c37f58da451fb/static/app/components/core/menuListItem/index.chonk.tsx#L52